click_odoo_contrib/initdb.py

```diff
@@ -408,9 +451,9 @@ def main(
         click.echo(click.style(msg, fg="yellow"))
         return
     module_names = [m.strip() for m in modules.split(",")]
-    if not cache:
+    if not cache :
         if new_database:
-            odoo_createdb(new_database, demo, module_names, False)
+                odoo_createdb_without_cache(new_database, demo,module_names, lang, password, login, country, phone)
```

Comment:
Actually the cache options, controls the caching of database templates. This apparently merely disables _save_installed_checksums, which I think is good to preserve in this branch too.

Actually, there should be no reason to not cache in this case, and this should be doable easily by including lang, password, etc in the `addons_hash()` function, like the `demo` flag is already handled.

Reply:
Note that phone was not supported in older Odoo versions ?